server/core/Cargo.toml

```diff
@@ -28,6 +28,8 @@ axum-auth = "0.7.0"
 axum-extra = { version = "0.9.4", features = ["cookie"] }
 axum-macros = "0.4.1"
 axum-server = { version = "0.7.1", features = ["tls-openssl"] }
+base64 = "0.22"
```

Comment:
Should either be from workspace, or more likely, use the serde_as base64 wrappers.